server/service/src/invoice/inventory_adjustment/adjust_existing_stock/insert.rs

```diff
@@ -107,7 +107,9 @@ mod test {
     use util::inline_edit;
 
     use crate::{
-        invoice::inventory_adjustment::{insert::InsertInventoryAdjustment, AdjustmentType},
+        invoice::inventory_adjustment::{
+            adjust_existing_stock::InsertInventoryAdjustment, AdjustmentType,
```

Comment:
Sorry about these... had to move existing "insert inventory adjustment" to `adjust_existing_stock` to distinguish from this "add new stock" adjustment